bin/xym

```diff
@@ -22,8 +23,8 @@ def run():
     parser.add_argument(
         "--strict", type=bool, default=False,
         help='Optional flag that determines syntax enforcement; '
-        "'If set to 'True', the <CODE BEGINS> / <CODE ENDS> "
-        "tags are required; default is 'False'")
+             "'If set to 'True', the <CODE BEGINS> / <CODE ENDS> "
```

Comment:
In general, please try not to make unnecessary whitespace changes. They add to the noise when reviewing PRs.